feat(content): Add three more human starts (#11302)
endless-sky/endless-sky: data/starts.txt

	set "license: Pilot's"
	set "species: human"
	set "human space start"
	set "start: default"

# Patch for if the player started a game before 0.10.17
mission "Start: Default Patch"
	invisible

> 	landing

Reviewer: ```suggestion
	invisible
	landing
	non-blocking
```

	`	Your father brings you to the Republic Department of Starship Licensing building in the spaceport. The lines are long, but your father walks right up to one of the desks. Before the person in the front of the line or the one behind the desk can complain, he flicks a credit chip at each of them - both worth a substantial amount. "I'd like to get a pilot's license for my child."`
	`	"I'm sorry," the person behind the desk starts saying, "but you'll need to go through the process of applying through the net first, then wait three to five business days before-"`
	`	Your father tosses another credit chip at the person behind the desk, prompting them to hand you a stylus and a pad. "Sign here, please. Your pilot's license will be here in a moment."`
	`	Without any hesitation and after skipping past a number of pages of legal text, you sign your name:`
	name
	`	You enter the shipyard, your brand new pilot's license held tight in one hand and 500,000 credits from your father in the other. There are almost a dozen ship models for sale, but there are only four that you are able to afford. Which one you choose will determine your future...`

start "syndicate"
	name "Escaping Squalor"

> 	description `You grew up on Earth, an overpopulated world with few opportunities. But you took a contract working for the Syndicate, and they gave you a place to live, enough to eat, and a decent wage off of Earth. Afterward you moved to Antipode, a cheap frontier world, but found that it wasn't any better than Earth. Fed up, you used what you had left to travel to the closest planet with a shipyard...`

Reviewer 2: Probably want to explicitly mention that you're going from Antipode to Delve to avoid confusion.
```suggestion
	description `You grew up on Earth, an overpopulated world with few opportunities. But you took a contract working for the Syndicate, and they gave you a place to live, enough to eat, and a decent wage off of Earth. Afterward you moved to Antipode, a cheap frontier world, but found that it wasn't much better than Earth. Fed up, you used what you had left to travel to Delve, the closest planet with a shipyard...`
```

	scene "scene/icy opening"
	`Delve is unlike any planet you've ever been on: the entire spaceport is built into a colossal mountain cave, one side exposed to the open air. The beams and buildings making up the port dwarf the pitiful structures of Antipode, yet feel distinct from the crushing urban landscapes of Earth.`
	`	You hand the last of your credits to the captain that brought you here. Before she leaves, she gives you some advice. "Just a heads up: if you don't feel like dying early, head north after launching. South of here is nothing but uninhabited systems and pirates. You wouldn't last long in any ship you could afford." You thank her for the ride and advice before parting ways.`
	`	You stop by the shipyard first to see what they have. You're noticed by a salesman who seems reluctant to approach you, perhaps due to the shabbiness of your clothes compared to those around you. You ask him what it would take to purchase a ship here. He mentions the necessity of a pilot's license before beginning his spiel about all the cheapest models, but you cut him short and say that's all you needed.`

> 	`	The bank is luckily not very busy at the moment. One of the tellers eyes you up as you enter. "What can we help you with?" he asks with unhidden glee. You say how you're in need of a loan to buy a starship; 600,000 credits should be enough to cover the cost of the license and the ship. After handing over your Syndicate Account information, the teller comes back with a piece of paper. This whole process seems suspiciously streamlined to you.`

Reviewer 2: Bit of phrasing cleanup, and couple other tweaks. In the Syndicate Business chain, financial transactions require a biometric and is overseen by Syndicated Systems, so added the biometric & "Systems" for some continuity.

```suggestion
	`	Luckily, the bank isn't very busy at the moment. One of the tellers sizes you up as you enter. "What can we help you with?" he asks with unrestrained glee. You say you're in need of a loan to buy a starship; 600,000 credits should be enough to cover the the license and the ship. After you hand over your Syndicated Systems Account information and scan your fingerprint, the teller returns with a piece of paper.`
```

> 	`	"Lucky for you, rates are great right now!" You glance at the paper: the daily interest rate is 0.5%, meaning the daily payments are 3,580 credits for a whole year. If you don't pay off any of your loan early, you'll have paid a 706,571 credits in interest, more than the 600,000 loan itself. "Just sign here." The teller grins.`

Reviewer 3: ```suggestion
	`	"Lucky for you, rates are great right now!" You glance at the paper: the daily interest rate is 0.5%, meaning the daily payments are 3,580 credits for a whole year. If you don't pay off any of your loan early, you'll have paid 706,571 credits in interest, more than the 600,000 loan itself. "Just sign here." The teller grins.`
```

> 	`	"Lucky for you, rates are great right now!" You glance at the paper: the daily interest rate is 0.5%, meaning the daily payments are 3,580 credits for a whole year. If you don't pay off any of your loan early, you'll have paid a 706,571 credits in interest, more than the 600,000 loan itself. "Just sign here." The teller grins.`

Reviewer: ```suggestion
	`	"Lucky for you, rates are great right now!" You glance at the paper: the daily interest rate is 0.5%, meaning the daily payments are 3,580 credits for a whole year. If you don't pay off any of your loan early, you'll have paid 706,571 credits in interest, more than the 600,000 loan itself. "Just sign here." The teller grins.`
```

> 	`	"Lucky for you, the interest rates are great right now!" You take a glance at the paper and feel a sense of dread fall over you. The daily interest rate is 0.5%, meaning the daily payments are 3,580 credits for a whole year. If you don't pay off any of your loan early, you'll have paid a disgusting 706,571 credits in interest. "Just sign here." The teller grins.`

Reviewer 2: Let the player experience dread for themselves on seeing the interest total.

```suggestion
	`	"Lucky for you, rates are great right now!" You glance at the paper: the daily interest rate is 0.5%, meaning the daily payments are 3,580 credits for a whole year. If you don't pay off any of your loan early, you'll have paid a 706,571 credits in interest, more than the 600,000 loan itself. "Just sign here." The teller grins.`
```

		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
	scene "scene/train"

> 	`The train ride from your hometown to the capital is calm and without incident. Several seats forward, a young couple close to your age discuss the down payment on an apartment as their legs nudge their luggage. Behind you sits one of the only other passengers in the car; a much older man in greasy overalls, his gray hair makes you suspect he's from off-world.`

Reviewer 4: ```suggestion
	`The train ride from your hometown to the capital is calm and without incident. Several seats forward, a young couple close to your age discuss the down payment on an apartment as their legs nudge their luggage. Behind you sits one of the only other passengers in the car, a much older man in greasy overalls. His gray hair makes you suspect he's from off-world.`
```

> 	`	Before long, you arrive in New Reykjavik and find the government office responsible for handling starship loans. The place is relatively empty compared to the photos you'd seen online a week prior; most applicants have already been processed in the week after exam results were posted. The only people left are those sick at the time, those too lazy to apply in a timely manner, and those willing to wait if it means avoiding a mosh pit of students in a government building. You hand over your application papers, and after verifying them against online records, the office worker hands you two cards. One of them is the card for applying for a loan at a nearby bank; the other is your pilot's license, fully paid for.`

Reviewer 2: Something like a "Certificate of Eligibility" or "Pre-approval Certification" is more commonly used to describe a document that confirms eligibility for a low-cost loan. "The only people left" sentence also felt a bit verbose, and other tweaks were mostly phrasing.

```suggestion
	`	Before long, you arrive in New Reykjavik and find the government office responsible for handling starship loans. The place is relatively empty compared to the photos you'd seen online a week prior. Most applicants have already come and gone in the week since exam results were posted. The only people left are those who were too sick or lazy to come by last week, as well as those wishing to avoid a mosh pit of students in a government building. You hand over your application, and after verifying them against online records, the office worker hands you two cards. One of them is your Certificate of Eligibility for a low-interest loan at a nearby bank; the other is your pilot's license, paid in full.`
```

	`	The bank is only a few blocks away from the office. You insert the card into an automatic clerk, and after a soft ping, a large document fills its screen.`
	`	As you scroll through, a synthesized voice says, "You are borrowing 580,000 credits to be repaid over the course of a year. Your daily interest rate is 0.2%, which means that your daily payments are 2,241 credits, and by the end of the year you will have paid 237,784 credits in interest. Please sign your name on the screen when you are ready."`
	`	After carefully rereading every page of the loan agreement, you sign your name:`
	name

> 	`	Eager to get started, you rush out of the bank, hop on the metro, and head towards the shipyard. You can barely keep your face straight as the rest of the carriage wear blank expressions. After getting off, you consider throwing away your planetary transport pass, but decide against it.`

Reviewer 2: This might be breaking the "don't tell me what I'm thinking rule."

```suggestion
	`	Eager to get started, you rush out of the bank, hop on the metro, and head towards the shipyard. You can barely keep a straight face as the rest of the carriage wear blank expressions. After getting off, you consider throwing away your planetary transport pass, but decide against it.`
```

Author: It's no more egregious than the description of the player's expression in the default start.

> 	`	You had seen videos of the shipyard in New Reykjavik, but visiting the site in person blows your wildest dreams out of the water. Throughout the gleaming-white facility, salespeople show of a variety of holographically-rendered vessels to sharply-dressed clients, while skyscraper-sized ships loom silently in the background. Although most of the ships are far outside of your price range, you spot a few that you'll be able to purchase. Which one you choose will determine your future...`

Reviewer 2: ```suggestion
	`	You had seen videos of the shipyard in New Reykjavik, but visiting the site in person blows your wildest dreams out of the water. Throughout the gleaming-white facility, salespeople show of a variety of holographically-rendered vessels to sharply-dressed clients, while skyscraper-sized ships loom silently in the background. Although most of the ships are far outside of your price range, you spot a few lie within reach. The one you choose will shape your future...`
```

Author: "Which one you choose [...]" is the wording used in the default start, as well as all the other starts in this PR.

start "paradise"
	name "Birthday Surprise"

> 	description `You grew up on Mainsail, an oceanic Paradise World terraformed to perfection. Only the richest live on such worlds; luckily, your father is a wealthy financier. But you've always found the rich life boring and are more interested in living a life among the stars. As your eighteenth birthday party winds down, your father takes you aside to talk...`

Reviewer 2: ```suggestion
	description `You grew up on Mainsail, an oceanic Paradise World terraformed to perfection. Only the richest live on such worlds; luckily, your father is a wealthy financier. But you've always found your pampered life boring, and are more interested in living a life among the stars. As your eighteenth birthday party winds down, your father takes you aside to talk...`
```

	description `This start has has no mortgage, leaving you free to explore the galaxy at your own pace.`
	thumbnail "scene/mainsail"
	date 16 11 3013
	system "Alhena"
	planet "Mainsail"
	conversation "paradise intro"
	account

> 		credits 490000

Reviewer 2: Small thing, but I feel like a wealthy and cavalier father would just give a round 500,000 credits as a gift rather than a more specific 490,000.
```suggestion
		credits 500000
```

		score 600
	set "license: Pilot's"
	set "species: human"
	set "human space start"
	set "start: paradise"

conversation "paradise intro"
	action
		event "paradise descriptions" 0

> 		log "Got a pilot's license for my birthday, as well as 490,000 credits to purchase a starship. The galaxy is now at my fingertips."

Reviewer 4: ```suggestion
		log "Got a pilot's license for my birthday, as well as 500,000 credits to purchase a starship. The galaxy is now at my fingertips."
```

			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
	scene "scene/reception"
	`"I know you've wanted to be the captain of a starship for a long time now," your father says. "So as my birthday gift to you, how about we go get you a pilot's license and a ship from the shipyard?" You say yes without a second thought.`

> 	`	Your father brings you to the Republic Department of Starship Licensing building in the spaceport. The lines are long, but your father walks right up to one of the desks. Before the person in the front of the line or the one behind the desk can complain, he tosses a credit chip worth a substantial amount at each of them. "I'd like to get a pilot's license for my child."`

Reviewer 2: Prevent duplication of toss, and reordered for flow.
```suggestion
	`	Your father brings you to the Republic Department of Starship Licensing building in the spaceport. The lines are long, but your father walks right up to one of the desks. Before the person in the front of the line or the one behind the desk can complain, he flicks a credit chip at each of them - both worth a substantial amount . "I'd like to get a pilot's license for my child."`
```

> 	`	Your father brings you to the Republic Department of Starship Licensing building in the spaceport. The lines are long, but your father walks right up to one of the desks. Before the person in the front of the line or the one behind the desk can complain, he flicks a credit chip at each of them - both worth a substantial amount . "I'd like to get a pilot's license for my child."`

Reviewer 4: ```suggestion
	`	Your father brings you to the Republic Department of Starship Licensing building in the spaceport. The lines are long, but your father walks right up to one of the desks. Before the person in the front of the line or the one behind the desk can complain, he flicks a credit chip at each of them - both worth a substantial amount. "I'd like to get a pilot's license for my child."`
```

	`	"I'm sorry," the person behind the desk starts saying, "but you'll need to go through the process of applying through the net first, then wait three to five business days before-"`
	`	Your father tosses another credit chip at the person behind the desk, prompting them to hand you a stylus and a pad. "Sign here, please. Your pilot's license will be here in a moment."`
	`	Without any hesitation and after skipping past a number of pages of legal text, you sign your name:`
	name

> 	`	You enter the shipyard, your brand new pilot's license held tight in one hand and 490,000 credits from your father in the other. There are almost a dozen ship models for sale, but there are only three that you are able to afford. Which one you choose will determine your future...`

Reviewer 2: If you change the above to 500,000, close otherwise.
```suggestion
	`	You enter the shipyard, your brand new pilot's license held tight in one hand and 500,000 credits from your father in the other. There are almost a dozen ship models for sale, but there are only three that you are able to afford. Which one you choose will determine your future...`
```

start "syndicate"
	name "Escaping Squalor"

> 	description `You grew up on Earth, an overpopulated world with few opportunities. But you took a contract working for the Syndicate, and they gave you a place to live, enough to eat, and a decent wage off of Earth. Afterward you moved to Antipode, a cheap frontier world, but found that it wasn't any better than Earth. Fed up, you used what you had left to travel to the closest planet with a shipyard...`

Reviewer 2: Idea for this one that I'd be happy to write. After the start/intro, the player has the choice to sign another 6 month or 1 year contract as a merchant captain for the Syndicate. They get a modest salary and sign on-bonus, but 
1. Every month or two are called to stop whatever they're doing to do a cargo mission for the Syndicate.
2. And/or they need to do X number of Syndicate jobs per quarter.

If they fail their contract for whatever reason, they lose the salary and get a sizable fine.

This would show the Syndicate's role of offering work with a carrot/stick, optionally gives more structure to the first 6 months of play for those that want it, and can sprinkle in bread crumbs/setting for a future Syndicate campaign.

Author: Right now, adding brand-new start-specific missions and jobs (that aren't the tutorial missions) is beyond the scope of this PR.

Reviewer 2: Makes sense, feel free to resolve this one

	description `	This is a more difficult start, with larger daily repayments.`
	thumbnail "scene/earth"
	date 16 11 3013
	system "Scheat"
	planet "Delve"
	conversation "syndicate intro"
	account
		credits 480000
		score 200
		mortgage Mortgage

> 			principal 530000

Reviewer 2: Either this should be changed to 630000 or the below description should say 530,000.

		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
	scene "scene/tunnel"

> 	`Delve is unlike any planet you've ever been on: the entire spaceport is buried into a mountain, with one side open to receive incoming ships. The hallways leading from the hangars to the port proper are cramped, dirty, and covered in machinery, creating a sense of claustrophobia far different from the packed cities of Earth.`

Reviewer 2: When [talking with warp-core in a different PR about Delve](https://github.com/endless-sky/endless-sky/pull/11109#discussion_r1986101183), we came to the conclusion that Delve's spaceport is probably more open-air and less buried/cramped even for getting to the hangars (though that was what I initially thought as well). If we come to a differing conclusion here, we'll likely need to change the Delven Shipworm's blurb.

> 	`	You hand the last of your credits to the captain that brought you here. Before she leaves, she gives you some advice. "Just a heads up: if you don't feel like dying early, head north after launching. South of here is nothing but uninhabited systems and pirates. You wouldn't last long in whatever ship you're going to buy." You thank her for the ride and for the advice and part ways.`

Reviewer 2: ```suggestion
	`	You hand the last of your credits to the captain that brought you here. Before she leaves, she gives you some advice. "Just a heads up: if you don't feel like dying early, head north after launching. South of here is nothing but uninhabited systems and pirates. You wouldn't last long in any ship you could afford." You thank her for the ride and advice before parting ways.`
```

> 	`	You stop by the shipyard first to see what they have. You're noticed by a salesman who seems to reluctantly approach you, perhaps due to the fact that your clothes are rather shabby compared to those around you. You ask him what it would take to purchase a ship here. He mentions the neccessity of a pilot's license before beginning his spiel about all the cheapest models, but you cut him short and say that's all you needed.`

Reviewer 5: ```suggestion
	`	You stop by the shipyard first to see what they have. You're noticed by a salesman who seems to reluctantly approach you, perhaps due to the fact that your clothes are rather shabby compared to those around you. You ask him what it would take to purchase a ship here. He mentions the necessity of a pilot's license before beginning his spiel about all the cheapest models, but you cut him short and say that's all you needed.`
```

Reviewer 2: ```suggestion
	`	You stop by the shipyard first to see what they have. You're noticed by a salesman who seems reluctant to approach you, perhaps due to the fact that your clothes are rather shabby compared to those around you. You ask him what it would take to purchase a ship here. He mentions the necessity of a pilot's license before beginning his spiel about all the cheapest models, but you cut him short and say that's all you needed.`
```

> 	`	The bank is luckily not very busy at the moment. One of the tellers eyes you up as you enter. "What can we help you with?" he asks with a strange glee in his voice. You say how you're in need of a loan to buy a starship; 630,000 credits should be enough to cover the cost of the license and the ship. After handing over your Syndicate Account information, the teller comes back with a piece of paper. This whole process seems suspiciously streamlined to you.`

Reviewer 2: ```suggestion
	`	The bank, luckily, isn't very busy. One of the tellers gives you a once over as you enter. "What can we help you with?" he asks with a strange glee in his voice. You explain how you need a loan to buy a starship; 630,000 credits should be enough to cover the cost of the license and the ship. After handing over your Syndicate Account information, the teller comes back with a piece of paper. This whole process feels suspiciously streamlined.`
```

> 	`	"Lucky for you the interest rates are great right now!" You take a glance at the paper and feel a sense of dread fall over you. The daily interest rate is 0.5%, meaning the daily payments are 3,162 credits for a whole year. If you don't pay off any of your loan early, you'll have paid a disgusting 624,205 credits in interest. "Just sign here." The teller grins.`

Reviewer 5: ```suggestion
	`	"Lucky for you, the interest rates are great right now!" You take a glance at the paper and feel a sense of dread fall over you. The daily interest rate is 0.5%, meaning the daily payments are 3,162 credits for a whole year. If you don't pay off any of your loan early, you'll have paid a disgusting 624,205 credits in interest. "Just sign here." The teller grins.`
```

> 	`	The interest is terrible, but there's nothing else you can do about it. You sign your name:`

Reviewer 4: ```suggestion
	`	The interest is terrible, but there's nothing you can do about it. You sign your name:`
```
"nothing else" implies you've already tried, and failed, to do something.

	name

> 	`	After leaving the bank, you spend an hour waiting in line to apply for a pilot's license. When you finish going through the oddly easy process of applying for a license, you're left with 480,000 credits. You can't help but get the feeling that you're being played by the Syndicate once again in all this.`

Reviewer 2: Would it be more the gleeful banker & license office than the Syndicate though? If I applied for a loan and felt like the loan officer was oddly giddy and scammed me, I'd probably feel that the officer or office scammed me rather than something like "my nation is playing me." But this is probably telling the player too much of what they're feeling.

```suggestion
	`	After leaving the bank, you spend an hour waiting in line to apply for a pilot's license. The process is oddly simple, and by the time it's done, you're left with 480,000 credits. You can't shake the feeling that the Syndicate is playing you again.`
```

	set "Ask the Hai about the Unfettered: declined"

# Patch for if the player started a Hai space game before 0.10.17
mission "Start: Hai Patch"
	invisible

> 	landing

Reviewer: ```suggestion
	invisible
	landing
	non-blocking
```
